docs/user_guide.md

```diff
@@ -97,7 +97,7 @@ Project properties can be changed by clicking "Configuration".
 
 ## Managing members of a project  
 ### Adding members  
-You can add members with different roles to an existing project.  
+You can add members with different roles to an existing project. You can add a LDAP/AD user to project members under LDAP/AD authentication mode. 

```

Comment:
Approved this PR but this sentence may be rephrased to sound better. For example "when authentication mode is LDAP,  ......"